ios/Sources/SocialLoginPlugin/SocialLoginPlugin.swift

```diff
@@ -44,9 +44,11 @@ public class SocialLoginPlugin: CAPPlugin, CAPBridgedPlugin {
         }
 
         if let appleSettings = call.getObject("apple") {
-            if let appleClientId = appleSettings["clientId"] as? String,
```

Comment:
Why remove it? ClientID should be defined here not in login

Reply:
I removed because IOS doesn't use clientID. This variable is 100% useless for IOS.

Reply:
Ok for Iso but it shouldn’t be define in login for android

Reply:
I have no idea what you are talking about
IOS:
![image](https://github.com/user-attachments/assets/8655704d-e05e-4f4f-81e9-da60266bf225)

Android:
![image](https://github.com/user-attachments/assets/3e485a47-e590-40d2-898a-522bdd8312c1)